cli/minimal/Makefile

```
@@ -0,0 +1,31 @@
+# **************************************************************************************
+#
+#									Very minimal makefile.
+#
+# **************************************************************************************
+
+#
+#		Path / Name of emulator.
+#
+EMULATOR = emu_linux_x64
+#
+#		64tass optios.
+#
+ASMOPTS = -b -w -x
+
+#
+#		Assemble the source
+#
+assemble:
+	64tass test.asm $(ASMOPTS) -otest.xex -Ltest.lst
+#
+#		Run program without break.
+#
+run: assemble
+	x65emu -q file=test.xex	
```

We defined `EMULATOR` variable, so let's use it.

```suggestion
	$(EMULATOR) -q file=test.xex	
```